ripozo/decorators.py

```diff
@@ -201,6 +204,22 @@ def wrapped(cls, request, *args, **kwargs):
         return wrapped
 
 
+def assert_identity(cls, request):
+    identity_providers = getattr(cls, 'identity_providers', [])
```

Comment:
It feels weird to have `authentication_required` and `identity_providers` be the two entrypoints into this API. Have you considered standardizing around either the "authentication" or "identity" language?